src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -51,993 +34,148 @@ def tearDown(self):
-        self.assertFalse('cbsdId' in response)
-
-    # Load FSS database which includes at least one FSS site within 150 km of CBSD location registered above
-    self._sas_admin.InjectDatabase_url(config['fss_db'])
-
-    # Trigger daily activities
-    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
-
-    # Load GWBL database which includes at least one FSS site near the above registered CBSD location
-    self._sas_admin.InjectDatabase_url(config['gwbl_db'])
-
-    # Trigger daily activities and wait for it to get it complete
-    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
-    
-    # Request grant
-    config['grantRequests'][0]['cbsdId'] = cbsd_ids[0]
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
-    request = {'grantRequest': config['grantRequests']}
-
-    # Check grant response
-    response = self._sas.Grant(request)['grantResponse'][0]
-    self.assertEqual(response['cbsdId'], cbsd_ids[0])
-    self.assertIn(response['response']['responseCode'],config['expectedGrantResponseCodes'])
-    grant_id = response['grantId']
-    grant_expire_time = datetime.strptime(response['grantExpireTime'],
-                                          '%Y-%m-%dT%H:%M:%SZ')
-    del request, response
-   
-    # Relinquish grant1
-    request = {
-        'relinquishmentRequest': [{
-            'cbsdId': cbsd_ids[0],
-            'grantId': grant_id
-        }]
-    }
-    response = self._sas.Relinquishment(request)['relinquishmentResponse'][0]
-    # Check the relinquishment response
-    self.assertEqual(response['cbsdId'], cbsd_ids[0])
-    self.assertEqual(response['grantId'], grant_id)
-    self.assertIn(response['response']['responseCode'], config['expectedRelinquishResponseCodes'])
-    del request, response
-
-    # Load modified GWBL database
-    self._sas_admin.InjectDatabase_url(config['gwbl_db'])
-    
-    # Trigger daily activities
-    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
-
-    # Request grant
-    config['grantRequests'][0]['cbsdId'] = cbsd_ids[0]
-
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['lowFrequency'] = 3660000000
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['highFrequency'] = 3680000000
-    request = {'grantRequest': config['grantRequests']}
-
-    # Check grant response should be 400(INTERFERENCE)
+    config['grantRequest'][0]['operationParam']['operationFrequencyRange']['lowFrequency'] = freq_range[0]
+    config['grantRequest'][0]['operationParam']['operationFrequencyRange']['highFrequency'] = freq_range[1]
+    request = {'grantRequest': config['grantRequest']}
+    #Check grant response should be 400 (INTERFERENCE).
     response = self._sas.Grant(request)['grantResponse'][0]
     self.assertEqual(response['cbsdId'], cbsd_ids[0])
```

Comment:
Same comments here.

Reply:
The overlapping   frequency ranges are part of "default.config" file